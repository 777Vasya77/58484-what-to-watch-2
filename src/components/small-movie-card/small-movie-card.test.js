import React from 'react';
import renderer from 'react-test-renderer';
import SmallMovieCard from '~/components/small-movie-card/small-movie-card';

const FILM = {
  title: `Fantastic Beasts: The Crimes of Grindelwald`,
  image: `fantastic-beasts-the-crimes-of-grindelwald.jpg`
};

it(`SmallMovieCard component render correctly`, () => {
  const tree = renderer
    .create(
        <SmallMovieCard
          film={FILM}
          onCardLinkMouseEnter={() => {}}
        />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
