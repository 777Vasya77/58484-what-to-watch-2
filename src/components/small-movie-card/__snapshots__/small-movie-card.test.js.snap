// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallMovieCard component render correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <video
      height="175"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="/films#1"
    >
      Macbeth
    </a>
  </h3>
</article>
`;
